infra/lib/charts/app-of-apps.yaml

```diff
@@ -1,25 +0,0 @@
-apiVersion: source.toolkit.fluxcd.io/v1beta2
```

Comment:
Why was this and the three Typescript files in `infra/lib/charts/` deleted?

Reply:
We actually weren't using 2 of those 3 files. This app-of-apps was rolled into a hardcoded struct inside the Typescript